rfi_file_monitor/applicationwindow.py

```diff
@@ -490,6 +519,26 @@ def operations_button_cb(self, button):
         self._operations_box.pack_start(new_operation, False, False, 0)
         new_operation.show_all()
         self.update_monitor_switch_sensitivity()
+        self.controls_operations_live.append([_class.NAME, _class])
+        self.controls_operations_live_combo.set_model(self.controls_operations_live)
```

Comment:
I don't think it's necessary to set the model again: you already did so while creating the combobox. Updates to the model should be reflected automatically  in the combobox.

Reply:
I could not get this to work no matter what I tried. I tried to accesss the class.props.label but I just kept getting an error. Do you know how to set this?